microsoft/dotnet-framework-docker
# Add fix for Issue #406

**1. Add ngen update step in asp.net image‘s Dockerfile**
`&C:\Windows\Microsoft.NET\Framework64\v4.0.30319\ngen.exe update; `
`&C:\Windows\Microsoft.NET\Framework\v4.0.30319\ngen.exe update; `
**2. Add test to verify there is no pending assemblies**
- Add to ngen display asp.net's test images.
 `RUN C:\Windows\Microsoft.NET\Framework64\v4.0.30319\ngen.exe display`
- Verify that output log doesn't contains keyword StatusPending
```
var stdOut = ExecuteWithLogging($"build -t {tag}{targetArg}{buildArgsOption}{dockerfileArg} {contextDir}");
Assert.DoesNotContain(stdOut, "StatusPending");
```
